### ISSUE TITLE
named exports are not resolved relatively

### ISSUE THREAD

**Issue by TitanNano** [Galooshi/import-js#270]
i defined a few named exports
```JSON
    "namedExports" : {
        "af/util/make" : [ "Make", "hasPrototype", "Mixin"],
        "af/modules/DataBinding/Parser" : ["ObjectParser", "parseExpression", "assignExpression"]
    }
```
but when I import those functions the paths are not resolved. 
Lookup path is `src`, my module is in `src/af/modules/DataBinding/StyleBinding.js` and I'm importing e.g. `Make`. The path added by ImportJS is `af/util/make` and not `../../util/make`.

**Comment by lencioni** [comment]
Thanks for the report! My hunch is that this is related to #266. We'll look into it when we can, but in the meantime we'd be happy to review a PR that fixes this.

**Comment by trotzig** [comment]
I have a commit that I believe will resolve this: a8a3ce020cd65

Once that is merged on master and released (assuming no structural changes), there's one thing you need to alter in order to make relative paths work @TitanNano: 
add a `src` prefix to all named export paths (e.g. `"af/util/make"` becomes `"src/af/util/make"`). Without that change, there's no way for import-js to know what modules those paths resolve to.

**Comment by trotzig** [comment]
I've released 0.7.4 that includes the above fix. Let me know if that doesn't resolve your issue!

**Comment by TitanNano** [comment]
@trotzig are you going to update the atom package?

**Comment by trotzig** [comment]
Sorry, I forgot about that - thanks for the reminder! I just published `atom-import-js@v0.2.2` which includes these changes.

**Comment by TitanNano** [comment]
It works great now 👍
